### PR TITLE
Update documentation images for navigator

### LINKED ISSUE #13189: Update screenshots of Navigator
We have several screenshots of Navigator which are still based on older styles. Plus, over time, a few things changed here and there. It would be nice to collect instances in which we are showing Navigator in our documentation (both in Navigator's own page and others), take new screenshots depicting the same interaction and update those images.

### PR DESCRIPTION
**Pull request by fayi-da**
Fixes #13189

### CODE REVIEW

**Review by carrielaben-da — APPROVED** [review]
LGTM. Do the images have alt text associated?

**Review by ray-roestenburg-da — COMMENTED** [review]
Maybe you did this update just before https://github.com/digital-asset/daml/pull/14941 got merged, since there is a minor change there for the dropdown at the wrench icon?

**Comment by fayi-da** [comment]
@carrielaben-da Alt texts updated accordingly.

**Review by ray-roestenburg-da — APPROVED** [review]
LGTM

File: docs/source/tools/navigator/index.rst [hunk 1]
```diff
@@ -268,7 +268,7 @@ To exercise a choice:
 .. image:: images/choice-exercise.png
   :width: 85%
   :align: center
-  :alt: The Contract Details view with the Consolidate choice selected.
+  :alt: The Contract Details view with the Mergeb choice selected.
```
Comment (ray-roestenburg-da) [inline docs/source/tools/navigator/index.rst:271]: Minor typo: `Merge` instead of `Mergeb` @fayi-da

**Pull request merged**